style.css

```diff
@@ -399,3 +416,156 @@ footer a {
   color: #dbd8d7;
   font-size: 19px;
 }
+
+@media screen and (min-width: 768px) {
+  .logo {
+    margin-left: 140px;
+    border: 1px solid #679;
+  }
```

Comment:
- [ ] While I must commend the brilliant work you put in for this project, I suggest you adjust the styling starting at this media screen. from the screenshot below, you can see that the button overlaps with the text. seeing the awesome work you've done, please make the change and compliment the beautiful design 👏 

![image](https://user-images.githubusercontent.com/50990925/213362227-8844d664-330e-4f2a-8bbe-4bc9eb184c75.png)